schutzbot/deploy.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -euxo pipefail
 
+DNF_REPO_BASEURL=http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com
+
+# The osbuild-composer commit to run reverse-dependency test against.
+OSBUILD_COMPOSER_COMMIT=692a8076bb4bf38dae05ce99631ebcdd3f0ab054
```

Comment:
This is the reverse of the logic I introduced in https://github.com/osbuild/osbuild-composer/pull/1085. I was (after a lot of dithering) landing on keeping this sort of "intended to be updated" config in a separate file (`Schutzfile`). The only downside is that I needed to add `jq` in order to parse said file. But as we need to split the test image from the target and the jenkins runner anyway, maybe that is not such a big deal? I figured better to show the right idea for people to build on...

Either way, we should do the same across the repos.

Reply:
I really like your Schutzfile idea! It just wasn't fully fleshed out when I wrote this PR. Maybe land this now and do the change as a follow-up, after your PR lands?